### PR TITLE
Update faculty list of XJTU

### PR DESCRIPTION
**Pull request by yueshangzuo**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

The new PR for #4115.

### CODE REVIEW

File: csrankings-c.csv [hunk 1]
```diff
@@ -473,6 +473,7 @@ Chengwen Luo 0001,Shenzhen University,https://luochengwen.bitbucket.io,DPW5Pj8AA
 Chengxiang Zhai,Univ. of Illinois at Urbana-Champaign,http://czhai.cs.illinois.edu,YU-baPIAAAAJ
 Chengyu Song,Univ. of California - Riverside,https://www.cs.ucr.edu/~csong,EoypoXAAAAAJ
 Chengzheng Sun,Nanyang Technological University,http://www.ntu.edu.sg/home/czsun,eBpYUn4AAAAJ
+Chenhao Lin,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/zh/web/linchenhao/cv,YK0G990AAAAJ
```
Comment (emeryberger) [inline csrankings-c.csv]: Research fellow; not eligible for inclusion - please remove.

File: csrankings-d.csv [hunk 2]
```diff
@@ -912,6 +913,7 @@ Devorah Kletenik,CUNY,http://www.sci.brooklyn.cuny.edu/~kletenik,cYTgpmEAAAAJ
 Dexter Campbell Kozen,Cornell University,http://www.cs.cornell.edu/~kozen,LV1qWjgAAAAJ
 Dexter Kozen,Cornell University,http://www.cs.cornell.edu/~kozen,LV1qWjgAAAAJ
 Deying Li,Renmin University of China,http://info.ruc.edu.cn/academic_professor.php?teacher_id=60,NOSCHOLARPAGE
+Deyu Meng,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/web/dymeng,an6w-64AAAAJ
```
Comment (emeryberger) [inline csrankings-d.csv]: Math - not clear how this person meets the inclusion criteria. Please clarify.

File: csrankings-j.csv [hunk 3]
```diff
@@ -895,6 +895,7 @@ Jian Lv 0001,Nanjing University,http://cs.nju.edu.cn/58/2a/c2639a153642/page.htm
 Jian Ma 0004,Carnegie Mellon University,http://www.cs.cmu.edu/~jianma,kDZcBhkAAAAJ
 Jian Pei,Simon Fraser University,http://www.cs.sfu.ca/~jpei,zIMEVKsAAAAJ
 Jian Peng 0001,Univ. of Illinois at Urbana-Champaign,http://jianpeng.web.engr.illinois.edu,4wcAVXAAAAAJ
+Jian Sun 0009,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/web/jiansun/main,SSgNWOMAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv]: Math, not CS. As above.

File: csrankings-j.csv [hunk 4]
```diff
@@ -1188,6 +1191,7 @@ Jirí Kléma,Czech Technical University,https://ida.fel.cvut.cz/klema,zuwJZmEAAA
 Jiri Matas,Czech Technical University,http://cmp.felk.cvut.cz/~matas,EJCNY6QAAAAJ
 Jiwon Seo,Hanyang University,http://bdsl.hanyang.ac.kr/members/jiwonseo,O1WC6TAAAAAJ
 Jiyao An,Hunan University,http://csee.hnu.edu.cn/people/anjiyao,BeWWwfsAAAAJ
+Jizhong Zhao,Xi'an Jiaotong University,https://scholar.google.com/citations?user=CbGjM58AAAAJ,CbGjM58AAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv]: Google Scholar pages are not suitable as home pages. Please replace.

File: csrankings-t.csv [hunk 5]
```diff
@@ -486,6 +486,7 @@ Tie Luo,Missouri S&T,https://tluocs.github.io,4F60wpoAAAAJ
 Tie-Jun Zhao,Harbin Institute of Technology,http://homepage.hit.edu.cn/zhaotiejun,NOSCHOLARPAGE
 Tiejun Huang,Peking University,http://www.idm.pku.edu.cn/~tjhuang,knvEK4AAAAAJ
 Tiejun Zhao,Harbin Institute of Technology,http://homepage.hit.edu.cn/zhaotiejun,NOSCHOLARPAGE
+Tieliang Gong,Xi'an Jiaotong University,https://scholar.google.com/citations?user=yhsnUnMAAAAJ,yhsnUnMAAAAJ
```
Comment (emeryberger) [inline csrankings-t.csv]: Google Scholar != website, please replace.

File: csrankings-w.csv [hunk 6]
```diff
@@ -136,6 +136,7 @@ Wei Wang 0028,Nanjing University,http://cs.nju.edu.cn/wangwei,1y6wxQMAAAAJ
 Wei Wang 0030,HKUST,https://www.cse.ust.hk/~weiwa,NOSCHOLARPAGE
 Wei Wang 0054,University of Texas at San Antonio,http://www.cs.utsa.edu/~wwang,NOSCHOLARPAGE
 Wei Wu 0008,Beihang University,http://scse.buaa.edu.cn/info/1078/2642.htm,NOSCHOLARPAGE
+Wei Xi,Xi'an Jiaotong University,https://scholar.google.com/citations?user=Xt3XP-EAAAAJ,Xt3XP-EAAAAJ
```
Comment (emeryberger) [inline csrankings-w.csv]: Google Scholar != website, please replace.

File: csrankings-x.csv [hunk 7]
```diff
@@ -119,6 +119,7 @@ Xiaogang Ma,University of Idaho,http://www.uidaho.edu/engr/departments/cs/our-pe
 Xiaogang Wang 0001,Chinese University of Hong Kong,http://www.ee.cuhk.edu.hk/~xgwang,B5JgjsAAAAJ
 Xiaohao Cai,University of Southampton,https://www.ecs.soton.ac.uk/people/xc1f20,MOjShggAAAAJ
 Xiaohong Gao,Middlesex University,http://www.mitime.org,D4AqjAcAAAAJ
+Xiaohong Guan,Xi'an Jiaotong University,,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-x.csv]: Needs a home page.

File: csrankings-x.csv [hunk 8]
```diff
@@ -174,6 +175,7 @@ Xiaoming Zhang,Beihang University,http://scse.buaa.edu.cn/info/1038/3052.htm,NOS
 Xiaong-Dong Li,RMIT University,https://titan.csit.rmit.edu.au/~e46507,AQewL04AAAAJ
 Xiaoning Ding,NJIT,https://web.njit.edu/~dingxn,2lfRQF8AAAAJ
 Xiaoou Chen,Peking University,http://www.icst.pku.edu.cn/audioLab/researchgroup.htm,NOSCHOLARPAGE
+Xiaopeng Hong,Xi'an Jiaotong University,https://hongxiaopeng.com/,x3X-qysAAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: Now at HIT per home page; please fix.

File: csrankings-x.csv [hunk 9]
```diff
@@ -272,6 +276,7 @@ Xinyu Feng,Nanjing University,https://cs.nju.edu.cn/xyfeng,SuTEZ6EAAAAJ
 Xinyu Wang 0001,Zhejiang University,https://person.zju.edu.cn/en/wangxinyu_en,NOSCHOLARPAGE
 Xinyu Wang 0006,University of Michigan,https://web.eecs.umich.edu/~xwangsd,mvasb5AAAAAJ
 Xinyu Xing,Pennsylvania State University,https://ist.psu.edu/directory/faculty/uxx16,WBN9c6kAAAAJ
+Xinyu Yang,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/web/xyyang,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-x.csv]: Please confirm you have contacted DBLP, per instructions (this is a disambiguation page).

File: csrankings-x.csv [hunk 10]
```diff
@@ -245,6 +248,7 @@ Xinchao Wang,Stevens Institute of Technology,https://sites.google.com/site/sitex
 Xindong Wu 0001,University of Louisiana - Lafayette,http://www.ucs.louisiana.edu/~xxw8007,X8sHmqIAAAAJ
 Xinfeng Ye,University of Auckland,https://unidirectory.auckland.ac.nz/profile/x-ye,NOSCHOLARPAGE
 Xing Gao 0001,University of Delaware,http://xgao-work.github.io,Wkw_2-wAAAAJ
+Xing Wei,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/web/weixing,KNyC5EUAAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: Add suffix to disambiguate (see https://dblp.org/pid/14/4301.html).

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by yueshangzuo** [comment]
Sorry for the mistakes, errors you mentioned have been fixed (faculties with ambiguous DBLP IDs removed for now), also we verified other faculties, thanks.

File: csrankings-c.csv (hunk 1 above)
Comment (yueshangzuo) [reply csrankings-c.csv]: Removed

File: csrankings-d.csv (hunk 2 above)
Comment (yueshangzuo) [reply csrankings-d.csv]: Removed.

File: csrankings-j.csv (hunk 4 above)
Comment (yueshangzuo) [reply csrankings-j.csv]: Updated.

File: csrankings-j.csv (hunk 3 above)
Comment (yueshangzuo) [reply csrankings-j.csv]: Removed.

File: csrankings-t.csv (hunk 5 above)
Comment (yueshangzuo) [reply csrankings-t.csv]: Do not meet the inclusion criteria, removed.

File: csrankings-w.csv (hunk 6 above)
Comment (yueshangzuo) [reply csrankings-w.csv]: Updated.

File: csrankings-x.csv (hunk 7 above)
Comment (yueshangzuo) [reply csrankings-x.csv]: Updated.

File: csrankings-x.csv (hunk 8 above)
Comment (yueshangzuo) [reply csrankings-x.csv]: Verified and removed.

File: csrankings-x.csv (hunk 9 above)
Comment (yueshangzuo) [reply csrankings-x.csv]: Need to update, remove for now.

File: csrankings-x.csv (hunk 10 above)
Comment (yueshangzuo) [reply csrankings-x.csv]: Need to update, remove for now.

**Pull request merged**